src/containers/find-offer/offer-card-square/OfferCardSquare.styles.ts

```diff
@@ -38,14 +48,20 @@ export const styles = {
     right: 0
   },
   chipContainer: {
-    mb: '16px'
+    container: {
+      gap: '10px'
+    },
+    label: {
```

Comment:
move this styles to subjectLevelWithLabels styles
and use TypographyVariantEnum.Overline

Reply:
> move this styles to subjectLevelWithLabels styles and use TypographyVariantEnum.Overline


is this necessary if according to figma i need different styling on the OfferCardSquare?

Reply:
> move this styles to subjectLevelWithLabels styles and use TypographyVariantEnum.Overline

done that 👍